### PR TITLE
[13.0][MIG] product_analytic

### PR DESCRIPTION
**Pull request by ps-tubtim**

### CODE REVIEW

File: product_analytic/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    @api.onchange("product_id")
+    def _onchange_product_id(self):
+        res = super()._onchange_product_id()
+        inv_type = self.move_id.type
+        if self.product_id:
```
Comment (marylla) [inline product_analytic/models/account_move.py]: Hi there,
a few days ago a FIX-PR was made for version 12 in this line, see https://github.com/OCA/account-analytic/pull/283/

Can you add it here too? Thanks a lot! :)

Comment (ps-tubtim) [reply product_analytic/models/account_move.py]: @marylla Thank you. I fixed it.

**Comment by andres-pcg** [comment]
Hello there, can we have this merge already? Thanks

@marylla @OCA/accounting-maintainers @pedrobaeza

**Review by kittiu — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by rousseldenis** [comment]
@OCA/accounting-maintainers

**Pull request merged**